src/components/Article.jsx

```diff
@@ -1 +1,33 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getMonths } from '../models/constants/months';
+
+export function Article({ title, date, text }) {
+  const day = date.getDate();
+  const month = date.getMonth();
+  const year = date.getFullYear();
+
+  const dateName = day > 9 ? day.toString() : `0${day.toString()}`;
+  const monthName = getMonths()[month];
+  const formatDate = `${monthName} ${dateName}, ${year}`;
```

Comment:
simplify date formatting you have an example of how to do it in the initial code